src/test/java/net/revelc/code/impsort/ImpSortTest.java

```diff
@@ -201,4 +202,20 @@ public void testRemoveSamePackageImports() {
     assertTrue(imports.stream().anyMatch(imp -> "abcd.ef.Blah.Blah".equals(imp.getImport())));
     assertTrue(imports.stream().anyMatch(imp -> "abcd.efg.Blah2".equals(imp.getImport())));
   }
+
+  @Test
+  public void testResultStartWithComment() throws IOException {
+    Path p =
+            Paths.get(System.getProperty("user.dir"), "src", "test", "resources", "FirstImportComment.java");
+    Result result =
+            new ImpSort(StandardCharsets.UTF_8, eclipseDefaults, true, true, LineEnding.AUTO)
+                    .parseFile(p);
+
+    Path output = File.createTempFile("impSort", null).toPath();
+    result.saveSorted(output);
+
+    List<String> lines = Files.readAllLines(output);
+    assertTrue(lines.stream().filter(line -> line.equals(" * Some comment")).collect(Collectors.toList()).size() == 1);
```

Comment:
I'd be surprised if there wasn't a "Stream.count" method.

```suggestion
    assertEquals(1, lines.stream().filter(line -> line.equals(" * Some comment")).count());
```